Event addon migration

paulius-sladkevicius opened the pull request:
Related #20

gurneyalex committed:
First part of product migration analysis.
XML ID migration complete

gurneyalex committed:
fix missing dependency in product in v8

gurneyalex committed:
updated product migration analysis

gurneyalex committed:
first pass at product migration script

gurneyalex committed:
fix bad refactoring

gurneyalex committed:
compute active attribute of template from attribute of variants

gurneyalex committed:
copy product_product.image to product_product.image_variant

gurneyalex committed:
updated work analysis

gurneyalex committed:
migration of product.packaging and product.ul

guewen committed:
migration of product_template.cost_method, moved in stock_account

gurneyalex committed:
reinstall data from product moved to mrp

gurneyalex committed:
add migration for product.packaging to template

gurneyalex committed:
fix error in previous refactoring

gurneyalex committed:
Merge branch 'product-v8-migration' of https://github.com/guewen/OpenUpgrade into product-v8-migration

guewen committed:
Migrate product_template.standard_price.

Now a fields.property so the prices are moved to properties.
Also, entries for the prices will be created in product_price_history.

legalsylvain committed:
[ADD] check value for ir_ui_view.type field.

legalsylvain committed:
[REF] Remove useless field migration because 'oldname' is defined;

guewen committed:
Updated openupgrade_analysis_WORK.txt with comments

gurneyalex committed:
packaging field is o2m -> no colum renaming required

Commit:
Event addon migration

bwrsandman commented:
Forgot to say in #20:
Just update `openerp/openupgrade/doc/source/modules70-80.rst` to reflect the coverage of the module.
You will need to rebase with upstream/master

```
git remote add upstream https://github.com/OpenUpgrade/OpenUpgrade.git
git pull --rebase upstream master
```

gurneyalex committed:
fix packaging/ul migration

gurneyalex committed:
re-add the change on product consultant

gurneyalex committed:
added AGPL header

Commit:
Changed migration status of event module

paulius-sladkevicius commented:
@bwrsandman I've added it

gurneyalex committed:
fix typo

bwrsandman commented:
@komsas Thank you!
:+1:

StefanRijnhart commented on addons/event/migrations/8.0.0.1/openupgrade_analysis_work.txt:
```diff
@@ -0,0 +1,51 @@
+
+event        / event.event              / note (text)                   : DEL
+# Fields argument 'oldname' used, now points to 'description' field
+
```
Please convert to html using openerp.tools.mail.plaintext2html

paulius-sladkevicius replied:
@StefanRijnhart  Could you explain why it should be done?
If this manner everybody should do that? e.g. in the #9

paulius-sladkevicius replied:
Please ignore last questions.

guewen committed:
Merge remote-tracking branch 'openupgrade/master' into product-v8-migration

guewen committed:
openerp namespace is missing in the import

guewen committed:
Wrong API call (PR submitted upstream)

guewen committed:
Migrate 'variants' and 'price_extra' using the new attribute models

legalsylvain committed:
[IMP] 'base' analysis. now 'ir.actions.server' changes only are missing

guewen committed:
Merge remote-tracking branch 'gurneyalex/product-v8-migration' into product-v8-migration

Conflicts:
	addons/product/migrations/8.0.1.1/post-migration.py

guewen committed:
pep8, unused models

hbrunn committed:
[ADD] adapt migrate.py for version 8.0
[IMP] correct shebang and make executable

guewen committed:
Write a 'Unix time zero' on the product price history entries so product have a standard price for the past, not only from now

florentx committed:
[REF] openupgrade_records: sorting key can be a tuple

legalsylvain committed:
[ADD] finish analyses of 'base';
[ADD] post-migration files to manage invalid state value for ir_actions_server

hbrunn committed:
Merge branch '8.0' into migrate.py-8.0

hbrunn committed:
[IMP] update help for migrate.py

guewen committed:
pep8: missing whitespace around operator

legalsylvain committed:
[REF] Pep8

Commit:
Simplify migration calling, fix breakpoint use

Don't write to a temp file
Use `load_module` instead of `load_source`

This allows the use of breakpoints in IDEs and in pdb

Signed-off-by: Sandy Carter <sandy.carter@savoirfairelinux.com>

guewen committed:
Migrate product_template.cost_method to property (in stock_account)

hbrunn committed:
[IMP] link to head, not to specific branch

Commit:
Conversion of text field value to HTML

paulius-sladkevicius commented:
Added generic function that can convert text value to HTML.

bwrsandman commented:
@komsas nice function, but wouldn't it be more useful to put it in `openerp.openupgrade.openupgrade80`?

paulius-sladkevicius commented:
Firstly we should decide if it's worth to put there. Needs to check if there more places where it could be used.

Commit:
Meet PEP-8

bwrsandman commented:
@StefanRijnhart @hbrunn 

> wouldn't it be more useful to put `convert_field_to_html()` in openerp.openupgrade.openupgrade80?

legalsylvain committed:
Remove a line in order to be pep8 compliant after merge

arthru committed:
Merge branch 'base' of https://github.com/legalsylvain/OpenUpgrade into legalsylvain-base

Conflicts:
	openerp/addons/base/migrations/8.0.1.3/pre-migration.py

bwrsandman committed:
Merge pull request #61 from savoirfairelinux/travis

Update travis pep8 checking to cover migration files starting in pre- and pro-

arthru committed:
Merge pull request #47 from florentx/sort_key

[REF] openupgrade_records: sorting key can be a tuple

legalsylvain committed:
Merge pull request #60 from arthru/legalsylvain-base

Base migration finished, based on work from legalsylvain

Commit:
Merge pull request #37 from guewen/product-v8-migration

[ADD] Product v8 migration

legalsylvain commented:
Agree with @bwrsandman remarks.

hbrunn committed:
Merge pull request #50 from savoirfairelinux/breakpoints

Simplify migration calling, fix breakpoint use

arthru committed:
Update migration status for base

StefanRijnhart commented:
As it is already  refactored out, we might as well have it in the API. I would make that openupgrade.py (not openupgrade_80.py) because this is not particularly 8.0 specific.

Commit:
Merge remote-tracking branch 'hbrunn/migrate.py-8.0' into 8.0

paulius-sladkevicius commented:
@StefanRijnhart in such case I'll try to move it to API

hbrunn committed:
[IMP] don't look at models with _auto = False

legalsylvain committed:
Merge pull request #66 from hbrunn/ignore_nonauto

[IMP] ignore models with _auto = False

Commit:
Moved convert_field_to_html to API

Commit:
Merge branch '8.0' into event_migration

Conflicts:
	openerp/openupgrade/openupgrade.py

paulius-sladkevicius commented:
Sorry, looks by mistake I've merged 8.0 while resolving conflict, but this pull request points to master. I'll prepare a new clean pull request to 8.0.
